src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/InflaterManaged.cs

```diff
@@ -506,7 +507,7 @@ private bool DecodeBlock(out bool end_of_block_code_seen)
                         goto case InflaterState.HaveDistCode;
 
                     case InflaterState.HaveDistCode:
-                        // To avoid a table lookup we note that for distanceCode >= 2,
```

Comment:
Nope, no idea. It actually caused me a large amount of wasted time because I changed the code to be in line with the comment at first and faced a bunch of tricky new failures.